### PR TITLE
scripts: ceph project activity analysis tool

### PR DESCRIPTION
**Pull request by dachary**
Signed-off-by: Loic Dachary <loic@dachary.org>

### CODE REVIEW

**Comment by tchaikov** [comment]
@dachary [OT] can we add a "--fail-under" option for ceph-disk's coverage test before we have a 100% coverage for it?

File: src/script/ceph-fs-report/ceph_fs_report/main.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+            return False
+        created_at = time.strptime(created_at, self.TIME_TEMPLATE)
+        return created_at < releases[branch]
+
+    @staticmethod
+    def pr_p(number):
+        return 'https://github.com/ceph/ceph/pull/' + str(number)
+            
+    def is_selected_labels(self, labels, number):
+        components = set(['cephfs', 'core', 'common', 'documentation',
```
Comment (tchaikov) [inline src/script/ceph-fs-report/ceph_fs_report/main.py]: could you add "build/ops" to this list?

Comment (dachary) [inline src/script/ceph-fs-report/ceph_fs_report/main.py]: done

File: src/script/ceph-fs-report/ceph_fs_report/main.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                average += age * (float(pr_count) / total_pr_count)
+            average_age.append([day * one_day * 1000, int(average)])
+        print("merges_over_time = " + str(average_age[60:]))
+
+    def main(self):
+        self.set_cache()
+        self.set_github()
+        self.set_git()
+        self.fetch_prs(self.args.pr_limit)
+        self.print_ages()
```
Comment (tchaikov) [inline src/script/ceph-fs-report/ceph_fs_report/main.py]: @dachary `print_ages()` is not defined.

**Pull request closed without merging**